test/QueryPlannerTest.cpp

```diff
@@ -2981,3 +2981,53 @@ TEST(QueryPlanner, Exists) {
       "GRAPH ?g { ?u ?v ?c}}}",
       filter);
 }
+
+// _____________________________________________________________________________
+TEST(QueryPlanner, PropertyPathWithGraph) {
```

Comment:
This test case mixes "property path with graph" and "empty property path": ,
at least adjust the names.